pkg/libovsdb/notation.go

```diff
@@ -14,6 +17,8 @@ type Operation struct {
 	Where     []interface{}            `json:"where,omitempty"`
 	Until     string                   `json:"until,omitempty"`
 	UUIDName  string                   `json:"uuid-name,omitempty"`
+	Comment   string                   `json:"comment,omitempty"`
+	Durable   bool                     `json:"durable,omitempty"`
```

Comment:
according to the OVSDB extensions:
https://docs.openvswitch.org/en/latest/ref/ovsdb-server.7/#insert

> As an extension, Open vSwitch 2.13 and later allow an optional uuid member to specify the UUID for the new row. The specified UUID must be unique within the table when it is inserted and not the UUID of a row previously deleted within the transaction. If the UUID violates these rules, then the operation fails with a duplicate uuid error.